token/jwt/claims_id_token.go

```diff
@@ -87,9 +106,6 @@ func (c *IDTokenClaims) ToMap() map[string]interface{} {
 		ret["amr"] = c.AuthenticationMethodsReference
 	}
 
-	ret["iat"] = float64(c.IssuedAt.Unix())
```

Comment:
So `iat` in ID tokens was always added but in access tokens only if set.

I made it so that all fields are not set if empty. I think this is cleaner.